utils.groovy

```diff
@@ -10,6 +10,8 @@ PROTECTED_JENKINSES = [
         ['fcos-builds', 'prod/streams/${STREAM}', false],
     'https://jenkins-rhcos-devel.apps.ocp-virt.prod.psi.redhat.com/':
         ['rhcos-ci', 'prod/streams/${STREAM}', true],
+    'https://jenkins-rhcos--runtime-int.apps.int.preprod-stable-spoke1-dc-iad2.itup.redhat.com/':
+        ['rhcos-ci', 'prod/streams/${STREAM}', true],
```

Comment:
CI is complaining about duplicate list literals here. We only have the `rhcos-ci` bucket for devel, so I had to re-use it until we migrate completely away from the ocp-virt cluster. I feel like this _shouldn't_ be a problem because, while the values are the same, the keys are indeed unique. 

Also, no issues reported while using this code to stand up devel in the ITUP cluster.